edx_exams/apps/core/models.py

```diff
@@ -20,8 +20,6 @@ class User(AbstractUser):
     full_name = models.CharField(_('Full Name'), max_length=255, blank=True, null=True)
     lms_user_id = models.IntegerField(null=True, db_index=True)
 
-    anonymous_user_id = models.IntegerField(null=True, db_index=True)
```

Comment:
@ashultz0 per your recommendation (thanks for sharing that doc), my plan is to separate this into two parts. 

1. In this PR, I'm dropping the anonymous_user_id int column based on the steps here: [Everything About Database Migrations](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations).
2. Then, I will follow the steps in the doc to add a new anonymous_user_id UUIDField with the default as a UUID. 

I decided to do it this way than doing an AlterField, as this seemed to be a little less safe and also we don't need the previous anonymous_user_id int field so there's no operation required here to copy over the values. Does this make sense/sound good?

Reply:
I think that would be my first instinct - just kill the old one and put a new one back in the same place

Reply:
but per the DB migrations doc I think you need to split the field removal and the column removal migration into two separate tiny releases

Reply:
Ah that's right! I misread that--I'll update!